include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -264,12 +264,12 @@ class cartesian_composition :
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    constexpr cartesian_composition() noexcept : base_t{} {}
```

Comment:
can you directly add documentation? 
also why aren't those noexcept?

Reply:
I would rather do this separately, some more things might need to be changed here.